test/graph_rewrites/config_for_grappler_test.cc

```diff
@@ -41,6 +41,7 @@ namespace testing {
 // etc.,etc.
 
 TEST(GrapplerConfig, RConfig1) {
+  ActivateNGraph();
```

Comment:
Why is the explicit enabling/disabling of nGraph required?

Reply:
It's to disable de-assigning of clusters.

Reply:
You can use `NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS` ?

Reply:
Where? To run the entire unit test suite? I'm not sure why this isn't being done already.